### PR TITLE
24221 vaccine scheduling error msgs

### PR DESCRIPTION
**Pull request by LanceGundersen**
## Description
Per VSP Content feedback in #24101 and the Design System, update error validation messages to be more descriptive for better accessibility.
image

### To Test
Walk through the covid vaccine workflow and verify error validations

## Testing done
Local, unit, and e2e

## Screenshots
image
image
image
image

## Acceptance criteria
- [x] Changes are behind the va_online_scheduling_cheetah feature flag
- [x] CONTENT - Error validation message on the Received Dose Screener page matches the [copy doc](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/products/health-care/covid-vaccine-distro/vaos-for-distribution-initiative/copy-docs/received-dose-screener.md#error-validation)
- [x] CONTENT - Error validation message on the Facility Selection (Choose a location) page matches the [copy doc](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/products/health-care/covid-vaccine-distro/vaos-for-distribution-initiative/copy-docs/facility-selection.md#error-validation)
- [x] CONTENT - Error validation message on the Clinic Selection (Choose a clinic) page matches the [copy doc](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/products/health-care/covid-vaccine-distro/vaos-for-distribution-initiative/copy-docs/clinic-selection.md#error-validation)
- [x] CONTENT - Error validation message on the Contact Information page matches the [copy doc](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/products/health-care/covid-vaccine-distro/vaos-for-distribution-initiative/copy-docs/contact-information.md#error-validation)

## Definition of done
- [x] All acceptance criteria are met
- [x] [Documentation/screenshots](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/appointments/va-online-scheduling/design/cheetah_flow.md) are updated

### CODE REVIEW

File: src/applications/vaos/covid-19-vaccine/components/ContactInfoPage.jsx [hunk 1]
```diff
@@ -46,12 +48,14 @@ const uiSchema = {
     ...phoneConfig,
     'ui:errorMessages': {
       ...phoneConfig['ui:errorMessages'],
-      pattern:
-        'Please enter a valid 10-digit phone number (with or without dashes)',
+      pattern: 'Please enter a 10-digit phone number (with or without dashes)',
```
Comment (jbalboni) [inline src/applications/vaos/covid-19-vaccine/components/ContactInfoPage.jsx:51]: I don't think removing the "valid" text was intentional in the ticket. I added that the other day (after this ticket was written, I think) to reflect that we do some validation that the phone number is a real number. I think we should keep the text we already have, but might be worth checking with Lauren to be sure.

File: src/applications/vaos/covid-19-vaccine/components/ContactInfoPage.jsx [hunk 2]
```diff
@@ -46,12 +48,14 @@ const uiSchema = {
...
     },
   },
   email: {
     'ui:title': 'Your email address',
+    'ui:errorMessages': {
```
Comment (jbalboni) [inline src/applications/vaos/covid-19-vaccine/components/ContactInfoPage.jsx:56]: Huh, it's weird that we're not currently using `emailConfig` here. I think we should fix that. We can set `email: emailConfig,` and that should be good.

File: src/applications/vaos/covid-19-vaccine/components/ContactInfoPage.jsx (hunk 1 above)
Comment (LanceGundersen) [reply src/applications/vaos/covid-19-vaccine/components/ContactInfoPage.jsx]: I checked with Lauren and corrected it to read with `valid`

File: src/applications/vaos/covid-19-vaccine/components/ContactInfoPage.jsx (hunk 2 above)
Comment (LanceGundersen) [reply src/applications/vaos/covid-19-vaccine/components/ContactInfoPage.jsx]: Fixed

**Review by jbalboni — APPROVED** [review]

**Pull request merged**